packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md

```diff
@@ -87,47 +87,47 @@ ok -> ua: 302 to redirect_uri
 @enduml
 -->
 
-## Account Linking and Just-in-Time Provisioning
+## Account Linking and Just-In-Time provisioning
 
-When you allow your users to sign in to your app using their choice of Identity Providers, you can use account linking to help create a unified view of your users within your org. Additionally, you can use Just-in-Time (JIT) provisioning to create a seamless experience for users that sign-in to your application for the first time using their credentials from another Identity Provider.
+Use account linking to help create a unified view of your users within your org. Also, you can use Just-In-Time (JIT) provisioning to create a seamless experience for users that sign in to your application for the first time using their credentials from another Identity Provider.
 
 ### Account Linking
 
-Users can use multiple Identity Providers to sign in, and Okta can link all of those profiles to a single Okta user. This is called account linking. If, for example, a user signs in to your app using a different Identity Provider than they used for registration, account linking can establish that the user owns both identities, allowing the user to sign in from either account.
+Users can use multiple Identity Providers to sign in, and Okta can link all of those profiles to a single Okta user. This is called account linking. For example, a user signs in to your app using a different Identity Provider than they used for registration. Account linking can then establish that the user owns both identities. This allows the user to sign in from either account.
 
 Account linking is configured at the Identity Provider level. When you create an Identity Provider, these values are set by default:
 
--   **Account Link Policy** &mdash; Automatic
--   **Match Against** &mdash; Okta Username
--   **IdP Username** &mdash; idpuser.email
+* **Account Link Policy**: Disabled
+* **Match Against**: Okta Username
+* **IdP Username**: idpuser.email
 
-With these defaults, when any validated OpenID Connect JWT is provided, Okta searches the Universal Directory for a user's profile where the `email` attribute value passed by the Identity Provider (the **IdP Username** value) matches the `username` attribute value (the **Match Against** value). If there is a match, then the user is linked by mapping the required, static `sub` claim provided in the JWT to that user.
+With these defaults, when any validated OpenID Connect JWT is provided, Okta searches the Universal Directory for a user's profile where the `email` attribute value passed by the Identity Provider (the **IdP Username** value) matches the `username` attribute value (the **Match Against** value). If there's a match, then the user is linked by mapping the required, static `sub` claim provided in the JWT to that user.
```

Comment:
The default values have changed now, so you might want to change the description of how. that behaves, or say if account linking policy is automatic this is how it behaves.